rdkit/Chem/Draw/MolDrawing.py

```diff
@@ -141,27 +126,23 @@ def _getOffsetBondPts(self, p1, p2, offsetX, offsetY, lenFrac=None):
 
     # now move a portion of the way along the line to the neighbor:
     frac = (1. - lenFrac) / 2
-    fracP1 = fracP1[0]+dx*frac,\
-             fracP1[1]+dy*frac
-
-    fracP2 = fracP1[0]+dx*lenFrac,\
-             fracP1[1]+dy*lenFrac
+    fracP1 = fracP1[0] + dx * frac, fracP1[1] + dy * frac
+    fracP2 = fracP1[0] + dx * lenFrac, fracP1[1] + dy * lenFrac
     return fracP1, fracP2
 
-  def _offsetDblBond(self, p1, p2, bond, a1, a2, conf, dir=1, lenFrac=None):
+  def _offsetDblBond(self, p1, p2, bond, a1, a2, conf, direction=1, lenFrac=None):
     perp, offsetX, offsetY = self._getBondOffset(p1, p2)
-    offsetX = offsetX * dir
-    offsetY = offsetY * dir
+    offsetX = offsetX * direction
+    offsetY = offsetY * direction
```

Comment:
Changed the keyword argument name. This function is only used here and no where else.